src/index.js

```diff
@@ -6,3 +6,33 @@ import "./index.scss";
 
 // install snippets styles
 import "highlight.js/styles/default.css";
+
+window.onload = () => {
+  const localeSeletor = document.getElementById("localeSelector");
+  localeSeletor.innerHTML = isZhCN() ? 'English' : "中文";
+  localeSeletor.addEventListener("click", onLocaleChange);
+};
+
+const onLocaleChange = () => {
+  const currentProtocol = `${window.location.protocol}//`;
+  const currentHref = window.location.href.substr(currentProtocol.length);
+  const pathname = window.location.pathname;
+
+  window.location.href =
+    currentProtocol +
+    currentHref.replace(pathname, getLocalizedPathname(pathname));
```

Comment:
You can replace this whole function with `window.location.assign(getLocalizedPathname(window.location.pathname));`

Reply:
Cool